### PR TITLE
Update Mac Readme and add LFS + Hardware Virtualisation instructions

### PR DESCRIPTION
**Pull request by ldunnplaymint**
resolves: #1058

### CODE REVIEW

**Comment by ldunnplaymint** [comment]
I would like @5p0rt5BEArD to check this one, mostly to sanity check the steps for OSX setup

**Review by chrisfarms — APPROVED** [review]
LGTM

**Comment by 5p0rt5BEArD** [comment]
I'm going to test this on the freshly restored Mac

**Comment by ldunnplaymint** [comment]
I have cleaned up the readme a bit in this PR since Farm's approved it, will need another tick before it gets merged

File: README.md [hunk 1]
```diff
@@ -34,45 +42,74 @@ Client will be available at locahost:3000
...
+  ```
+  git clone --recurse-submodules https://github.com/playmint/ds
+  ```
 
-Client will be available at localhost:3000
+### 3. Install Foundry
+- **Install Foundry:** Visit [Foundry](https://book.getfoundry.sh/getting-started/installation)
+
+### 4. Install Abigen
+- **Install Abigen:** Visit [Abigen](https://geth.ethereum.org/downloads)
```
Comment (5p0rt5BEArD) [inline README.md:91]:
```suggestion
- `brew tap ethereum/ethereum`
- `brew install ethereum`
```

File: README.md [hunk 2]
```diff
@@ -34,45 +42,74 @@ Client will be available at locahost:3000
...
+- **Clone the Repository:** Use the following command:
+  ```
+  git clone --recurse-submodules https://github.com/playmint/ds
+  ```
 
-Client will be available at localhost:3000
+### 3. Install Foundry
+- **Install Foundry:** Visit [Foundry](https://book.getfoundry.sh/getting-started/installation)
+
+### 4. Install Abigen
```
Comment (5p0rt5BEArD) [inline README.md:90]:
```suggestion
### 4. Install Ethereum Tools
```

File: README.md [hunk 3]
```diff
@@ -34,45 +42,74 @@ Client will be available at locahost:3000
...
+- **Install Abigen:** Visit [Abigen](https://geth.ethereum.org/downloads)
+
+### 5. Install NPM
```
[the remark this answers is not in the document]
Comment (5p0rt5BEArD) [reply README.md:93]: should also add `nvm use` to be

**Review by 5p0rt5BEArD — CHANGES_REQUESTED** [review]
A few changes - I can make these :)

**Comment by chrisfarms** [comment]
Screenshot 2024-02-19 at 13 59 06

that capitalisation inconsistency puts me on edge 😄

File: README.md [hunk 4]
```diff
@@ -4,120 +4,104 @@ Contracts, client libraries, and plugins for the on-chain Downstream game.
...
+(**_🖥 Windows_** The following should all be installed to WSL:)
+- make
+- gcc
+- node (use nvm to match correct version)
+- go (version go1.19.13 - similar versions may be fine)
+- forge (version 0.2.0)
+- ethereum tools (including solc and abigen)
 
 <details>
+<summary>Installation Instructions</summary>
```
Comment (chrisfarms) [inline README.md:87]: I feel like having this expandable section which is already within an exandable second is quite hard to follow .... maybe we should consider breaking out all this "help with installing dependencies" bit into it's own `deps.md` file and just link to it

File: README.md [hunk 5]
```diff
@@ -128,68 +112,92 @@ Please refer to the instructions below for setup guidance.
...
+
+Once you are done, you can either build everything again with `make dev` or you can just
+build the map by using the `make map` command.
 </details>
 
 <details>
-
 <summary>Building from source (for production)</summary>
 
+### Building from source (for production)
```
Comment (chrisfarms) [inline README.md:199]: I don't think this section really makes sense any more ... the command is wrong, and there's no real need for anyone to do a "production" build.

If we did think it's worth keeping ... we should change it to just say something like:

```
A [github actions workflow](https://github.com/playmint/ds/actions) will build
production container images tagged with the commit sha on every merge to main
```

Comment (5p0rt5BEArD) [reply README.md:199]: seems like a Playmint org level instruction anyway - I'll kill the section

File: README.md (hunk 4 above)
Comment (5p0rt5BEArD) [reply README.md:87]: I think a separate .md for the tool installation instructions would help yes

Is there a convention for readme.md across multiple files?

**Review by 5p0rt5BEArD — APPROVED** [review]
Approved after making my own edits

File: README.md (hunk 4 above)
Comment (chrisfarms) [reply README.md]: not that I'm aware of ... what you have done looks good to me 👍

**Review by chrisfarms — APPROVED** [review]
LGTM 

did notice there's another "fold within a fold" in the docker section for the troubleshooting ... that could probably be broken out to avoid that odd UX like before ... but seems super nitty, it looks good
